api/cpp/lib.rs

```diff
@@ -55,12 +55,13 @@ pub unsafe extern "C" fn slint_post_event(
     i_slint_core::api::invoke_from_event_loop(move || {
         let ud = &ud;
         event(ud.user_data);
-    });
+    })
+    .unwrap();
 }
 
 #[no_mangle]
 pub unsafe extern "C" fn slint_quit_event_loop() {
-    i_slint_core::api::quit_event_loop()
+    i_slint_core::api::quit_event_loop().unwrap();
```

Comment:
I considered returning `bool` in the C++ API, but i decided against it because it is better to panic than ignore the error, IMHO. (because the error will be ignored most likely)

Reply:
Will the panic cause an exception on the C++ side or does this just abort?

Reply:
Since this is a C ABI there's no exception throwing for C++ involved. This will abort. But if that happens (it shouldn't) then at the moment there's nothing for the programmer to handle.

Reply:
We compile with panic=abort, this is set in the top Cargo.toml

Anyway, this is irrelevant for this pr since we panic all over the place anyway in case of misuse and stuff

Reply:
Good point about C ABI :-)

But we should really fix slint to not abort mid-term.

Reply:
> But we should really fix slint to not abort mid-term

I don't know.  It depends what kind of error is it. Is it a programmer error or a recoverable error?  Programmer error are bugs and it's fine to abort.  In this case of this patch, it is a bit unclear if it is a programming error or a recoverable error.  All the the call site of these function just unwrap, currently.